src/services/db/review.ts

```diff
@@ -83,3 +90,45 @@ export const approvePullRequest = (
       },
     }
   )
+
+export const getComments = async (
+  siteName: string,
+  pullRequestNumber: number
+) => {
+  const rawComments = await axiosInstance
+    .get<RawComment[]>(`${siteName}/issues/${pullRequestNumber}/comments`)
+    .then(({ data }) => data)
+  return _.compact(
+    rawComments.map((rawComment) => {
+      try {
+        const commentData = JSON.parse(rawComment.body)
+        const { user, message } = commentData
+        return {
+          user,
+          message,
+          createdAt: rawComment.created_at,
+        }
+      } catch (e) {
+        // Not properly formatted comment, ignore
+        return null
+      }
+    })
+  )
+}
+
+export const createComment = async (
+  siteName: string,
+  pullRequestNumber: number,
+  user: string,
+  message: string
+) => {
+  const stringifiedMessage = JSON.stringify({
+    user,
+    message,
+  })
+  return axiosInstance.post<void>(
+    `${siteName}/issues/${pullRequestNumber}/comments`,
+    // NOTE: only create body if a valid description is given
+    { body: stringifiedMessage }
```

Comment:
this comment seems to be erroneous - could we remove it?

Reply:
https://github.com/isomerpages/isomercms-backend/pull/534/commits/886deb1e5d59886e0a4942270a9b7b632612a453